examples/swarm-setup.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	metrics "github.com/ipfs/go-libp2p/p2p/metrics"
+	tu "github.com/ipfs/go-libp2p/testutil"
+	ma "github.com/jbenet/go-multiaddr"
+	swarm "go-libp2p/p2p/net/swarm"
```

Comment:
just add `github.com/ipfs/` to the imports, and make sure that `GO15VENDOREXPERIMENT` is set to `1`. it *should* work. (i'll try it too)